packages/test/test-end-to-end-tests/src/test/gc/gcSweepAttachmentBlobs.spec.ts

```diff
@@ -1252,7 +1252,14 @@ describeCompat("GC attachment blob sweep tests", "NoCompat", (getTestObjectProvi
 						clientType: "interactive",
 					},
 				],
-				async () => {
+				async function () {
+					// This test is flaky on FRS. See ADO:7922 and ADO:7923
+					if (
+						provider.driver.type === "routerlicious" &&
+						provider.driver.endpointName === "frs"
+					) {
+						this.skip();
+					}
```

Comment:
```suggestion
				async function () { // I changed it from the arrow function to the normal function and this.skip(); still erroring
					// This test is flaky on FRS. See ADO:7922 and ADO:7923
					if (
						provider.driver.type === "routerlicious" &&
						provider.driver.endpointName === "frs"
					) {
						this.skip();
					}
```
The errors im getting depending on the type of function
`'this' implicitly has type 'any' because it does not have a type annotation` when its a normal function
Object is possibly 'undefined' when its an arrow function